packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -484,9 +484,11 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 			if (this.handler === undefined) {
 				throw new Error("Attempted to process an inbound signal without a handler attached");
 			}
+
 			this.handler.processSignal({
 				clientId: message.clientId,
 				content: JSON.parse(message.content as string),
+				targetClientId: message.targetClientId,
 			});
```

Comment:
This does not conform to exact optional properties typing. (We haven't been able to enable it per too many things to fix.)
Either the type for `ISignalMessageBase` should be `targetClientId?: string | undefined` (not preferred) or this should only be added when it is present in message.
To do the latter either we use `{ ...message, content: JSON.parse(message.content as string)}` since message is alread `ISignalMessage` or we build object and selectively add in targetClientId. I would go with the former. It potentially adds more of the data, but that seems fine.

Reply:
Do double check you is getting this object and make sure there isn't something funny that would happen if recipients started seeing:
 	type?: string;
	clientConnectionNumber?: number;
	referenceSequenceNumber?: number;
(seems odd that `type` was never passed thru)